modules/apps/29-fee/client/cli/tx.go

```diff
@@ -47,6 +47,31 @@ func NewRegisterPayeeCmd() *cobra.Command {
 	return cmd
 }
 
+// NewRegisterCounterpartyPayeeCmd returns the command to create a MsgRegisterCounterpartyPayee
+func NewRegisterCounterpartyPayeeCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "register-counterparty-payee [port-id] [channel-id] [relayer] [counterparty-payee] ",
+		Short:   "Register a counterparty payee address on a given channel.",
+		Long:    strings.TrimSpace(`Register a counterparty payee address on a given channel.`),
```

Comment:
[nit] unnecessary `strings.TrimSpace` call, this is just a regular string.

Reply:
I left this in from existing code and I think there's a good few of them around. Maybe we could open a follow up PR to remove them if you want, some general housekeeping.